analyzer/src/types/hir.rs

```diff
@@ -74,6 +74,12 @@ pub struct MethodCall {
     pub function_id: FunctionId,
 }
 
+#[derive(Clone, Debug, PartialEq)]
+pub struct Subprocess {
+    pub inner: Box<TypedExpr>,
+    pub awaited: bool,
```

Comment:
```suggestion
    pub detached: bool,
```
Then invert the bools

Reply:
All those processes are "detached" from the main one: they don't share the same execution thread. The fact that some are awaited and some aren't is the only difference here. Also, I prefer using a field that isn't inverted later not to add unnecessary code bloat.